deltachat-rpc-client/tests/conftest.py

```diff
@@ -173,33 +173,27 @@ def get_uid_by_message_id(self, message_id) -> str:
 class IdleManager:
     def __init__(self, direct_imap) -> None:
         self.direct_imap = direct_imap
-        self.log = direct_imap.account.log
```

Comment:
`account` does not have `log`, this code got copy-pasted from CFFI bindings. I have removed `log` references that failed, previously this code was not working at all as nobody used IDLE on direct IMAP with JSON-RPC.